doc.html

```diff
@@ -0,0 +1,1296 @@
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    <b>pbench-clear-results</b>
+                                                    <br>start with a clean slate.
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    <b>pbench-copy-results</b>
+                                                    <br>copy results to the results repo.
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    <b>pbench-move-results</b>
+                                                    <br>move the results to the results repo and delete them from the local host.
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    <b>pbench-edit-prefix</b>
+                                                    <br>change the directory structure of the results (see the <a href="#web">Accessing results on the web</a> section below for details).
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    <b>XXX pbench-cleanup</b>
+                                                    <br>clean up the pbench run directory - after this step, you will need to register any tools again.
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    pbench-register-tool-set, pbench-register-tool and pbench-unregister-tool can also take a --remote option (see What does --remote do?) in <a href="learn.html">FAQ section</a>
+                                                    in order to allow the starting/stopping of tools and the postprocessing of results on multiple remote hosts.
+                                                </p>
+                                                <p class="info_text" style="text-align:justify">
+                                                    There is a set of miscellaneous tools for doing various and sundry things - although the name of the script indicates its purpose, if you want more information on these, you will have to read the code:
```

Comment:
Ah, open source documentation: if you want to know, read the code!

Reply:
@dbutenhof What exactly needs to be done here? should I ask them to go through open-source documentation instead of reading code?

Reply:
Sorry; that was not a request for changes, just a comment, which I probably should have kept to myself.